src/d3d9/d3d9_device.cpp

```diff
@@ -5551,12 +5563,18 @@ namespace dxvk {
       else if (unlikely(sampleCount != rtImageInfo.sampleCount))
         continue;
 
+      if (m_state.renderStates[ColorWriteIndex(i)] == 0)
```

Comment:
!

Reply:
Why `!`? We want to skip the RT if the color write mask is 0.

Reply:
Ooooh, you don't want the explicit `== 0`.